docs/networking.md

```diff
@@ -1,10 +1,18 @@
 # Kubernetes Networking Options
 
+## Introduction
+
+Kubernetes has a networking model in which Pods and Services have their own IP
+addresses. As Pods and Services run on servers with their own IP addresses and
+networking, the Kubernetes networking model is an abstraction that sits
+separately from the underlying servers and networks. A number of options,
+listed below, are available which implement and manage this abstraction.
```

Comment:
Some CNI providers don't utilize an overlay network so the pods share the servers' IP address space, making `...sits
separately from the underlying servers and networks` not necessarily true. I'm not sure if its worth making the distinction here though.

Reply:
If a CNI provider re-uses the servers' IP address space, then I think that's an implementation detail of the CNI provider itself. I think it's more pertinent here to explain why the user needs to make this choice to begin with, and that's because Kubernetes separates these concerns.